js/atomic-blocks.js

```diff
@@ -135,7 +135,8 @@
 			$('.search-drawer').attr('aria-expanded','false').slideUp(200);
 			$('.search-toggle span').hide();
 			$('.search-drawer-open .search-toggle i').toggle();
-        }
+			$('.site-header').toggleClass('search-drawer-open');
+		}
```

Comment:
Just eyeballing this out of context, but wouldn't we always want `removeClass` in a function called `closeDrawer()` to prevent the possibility that it opens it if called when already closed?

Reply:
Make's sense to me :-) 👍